packages/pwa-kit-extension-sdk/src/configs/babel/plugin-application-extensions.ts

```diff
@@ -51,7 +51,12 @@ module.exports = function replaceExtensionsPlaceholderContentPlugin({types: t}:
 
                 // Check if the file matches one of the files we want to replace
                 if (filePath.endsWith(extensionsPlaceholderFile)) {
-                    const newContent = renderTemplate(state.opts)
+                    const newContent = renderTemplate({
+                        ...(state.opts as ApplicationExtensionsLoaderOptions),
+                        configured: expand(state.opts.configured).map((extension) =>
+                            mergeConfigs(extension)
```

Comment:
I'm wondering is we could simply integrate the merging of the default into the `expand` util. I'm only saying that because using the word "merge" would typically imply that you have 2 or more arguments. So we can look at renaming this util, but then I got to think about building it into the expand, maybe even make it an optional action like `expand(extensionsConfig, {includeDefaultConfigs: true})`